sources/validation-api.asciidoc

```diff
@@ -2374,90 +2280,21 @@ acmeFactory.setSpecificConfiguration( [...] );
 
 [tck-testable]#[methodname]`getParameterNameProvider()` returns the [classname]`ParameterNameProvider` instance configured during the initialization of the [classname]`ValidatorFactory`.# It is particularly useful to build a [classname]`Validator` specific [classname]`ParameterNameProvider` wrapping the one from the [classname]`ValidatorFactory`.
 
-[classname]`ValidatorContext` returned by [methodname]`usingContext()` can be used to customize the state in which the [classname]`Validator` must be initialized. This is used to customize the [classname]`MessageInterpolator`, the [classname]`TraversableResolver`, [classname]`ParameterNameProvider` or the [classname]`ConstraintValidatorFactory`.
+[tck-testable]#[methodname]`getClockProvider()` returns the [classname]`ClockProvider` instance configured during the initialization of the [classname]`ValidatorFactory`.# It is particularly useful to build a [classname]`Validator` specific [classname]`ClockProvider` wrapping the one from the [classname]`ValidatorFactory`.
+
+[classname]`ValidatorContext` returned by [methodname]`usingContext()` can be used to customize the state in which the [classname]`Validator` must be initialized. This is used to customize the [classname]`MessageInterpolator`, the [classname]`TraversableResolver`, [classname]`ParameterNameProvider`, [classname]`ClockProvider` or the [classname]`ConstraintValidatorFactory`.
```

Comment:
Ah yes. I generally don't add it, here it was more c&p and staying consistent with the other usages in the same sentence.